homework7/Cat.java

```diff
@@ -0,0 +1,33 @@
+package org.hillel.homework7;
+
+import static org.hillel.homework7.Dog.getNumberOfDogs;
+
+public class Cat extends Animal{
+    private static final int MAX_RUN_DISTANCE = 200;
+    private static int numberOfCats = 0;
+
+    public Cat(String name, int age) {
+        super(name, "Cat", age);
+        numberOfCats++;
+    }
+
+    @Override
+    public void run(int distance) {
+        if (distance <= MAX_RUN_DISTANCE) {

```

Comment:
What will happen if `distance < 0` ? Would it be the correct input? How can we handle this corner case?